test cases/cmake/26 target file/meson.build

```diff
@@ -0,0 +1,37 @@
+project('cmake_code_gen', ['c', 'cpp'])
+
+# volker_inter_exe = executable('volker_inter_exe', 'volker_inter.cpp')
+# test_tgt = custom_target(
+#   'test_tgt',
+#   input : ['input1.txt'],
+#   output : ['output1.cpp'],
+#   command : [
+#     '/usr/bin/python3',
+#     volker_inter_exe.full_path(),
+#     ';;;'
+#   ],
+#   depends : [volker_inter_exe]
+# )
+
+# gen_src = custom_target('gen-output',
+#                         output : ['out.c'],
+#                         command : [volker_inter_exe, '@OUTPUT@'], install: true, install_dir: '.')
+
+# exe = executable('volker_exe', gen_src)
+x = 'abc' + 'def' + 'ghi'
+
+if meson.is_cross_build()
+    message('Test skipped, see https://github.com/mesonbuild/meson/pull/10607#issuecomment-1188117276 for an explanation.')
```

Comment:
This should be an `error('MESON_SKIP_TEST: explanation here')` probably? The test harness will then log it as `[SKIPPED]` instead of `[SUCCESS]`.

Reply:
Done